WanderLost/WanderLost/Client/Components/NotificationToggle.razor.css

```diff
@@ -0,0 +1,72 @@
+﻿.notification-toggle {
+}
+
+.notification-toggle__container {
+  display: flex;
+  margin-top: 4px;
+
+  align-items: center;
+
+  height: 42px;
+}
+
+.notification-toggle__input {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.notification-toggle__label {
+  display: block;
+  position: relative;
+
+  width: 42px;
+  height: 22px;
+
+  cursor: pointer;
+  text-indent: -9999px;
```

Comment:
?

Reply:
A way to hide label text so it's still accessible via screen readers. Not really needed here, but I decided to keep it anyway